tracer/src/Datadog.Trace/Iast/Location.cs

```diff
@@ -57,9 +66,25 @@ public Location(string? typeName, string? methodName, int? line, ulong? spanId)
 
     public int? Line { get; }
 
+    public string? StackId { get; }
+
     public override int GetHashCode()
     {
         // We do not calculate the hash including the spanId nor the line
         return IastUtils.GetHashCode(Path, Method);
     }
+
+    internal void ReportStack(Span? span)
+    {
+        if (span is not null && StackId is not null && _stack is not null && _stack.FrameCount > 0)
+        {
+#pragma warning disable CS8620 // Argument cannot be used for parameter due to differences in the nullability of reference types.
```

Comment:
tl;dr; we should _never_ have these AFAIK 🤔

Reply:
Some old TFMs are not null anotated